### PR TITLE
More analytics

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: guides/advanced_guides/configuration.md [hunk 1]
```diff
@@ -69,7 +69,21 @@ The master key allowing you to do everything on the server. If no master key is
...
+
+Analytics allow us to know how many users are using MeiliSearch and the following:
+
+- The application version.
+- The environment: development/production.
+- The number of days since the first start: Segment development/production.
+- The user email: (*Optional*) If the user wants to have alerts.
+- The server provider: (*Optional*).
+- The database size.
+- The last update.
```
Comment (qdequele) [inline guides/advanced_guides/configuration.md:82]:
```suggestion
- The last update time.
```

File: guides/advanced_guides/configuration.md [hunk 2]
```diff
@@ -69,7 +69,21 @@ The master key allowing you to do everything on the server. If no master key is
...
+- The number of updates per index.
```
Comment (qdequele) [inline guides/advanced_guides/configuration.md:83]:
```suggestion
- The number of updates.
```

File: guides/advanced_guides/configuration.md [hunk 3]
```diff
@@ -69,7 +69,21 @@ The master key allowing you to do everything on the server. If no master key is
...
+- The number of documents per index.
+
+**The index names are hashed.**
```
Comment (qdequele) [inline guides/advanced_guides/configuration.md:86]: We didn't retrieve the index names at all.

Comment (bidoubiwa) [reply guides/advanced_guides/configuration.md:86]:
```suggestion
```

**Pull request merged**